lib/codecs.js

```diff
@@ -3,7 +3,7 @@
 /* eslint-disable new-cap */
 
 var iconv = require('iconv-lite');
-var through = require('through2');
+var Transform = require('stream').Transform;
```

Comment:
We'll want this to be streamx

Reply:
@phated 
This `Tansform` operates text streams in text mode. while `streamx` operates binary streams in object mode.

Reply:
@sttk that's incorrect, streamx switches between object and binary automatically. you can receive/send either